.github/workflows/publish-next.yml

```diff
@@ -72,10 +78,10 @@ jobs:
         run: echo "::set-output name=version::$(amplify --version)"
       - name: Pull down AWS environments
         if: steps.environments-cache.outputs.cache-hit != 'true'
-        run: yarn environments auth pull
+        run: yarn environments auth pull:email
```

Comment:
build-and-runtime-test only needs the email environments

Reply:
Nice!!!